### PR TITLE
Dlukeomalley/overview tweaks

### PR DESCRIPTION
**Pull request by dlukeomalley**

### CODE REVIEW

File: docs/cli-usage.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# CLI Usage
```
Comment (dlukeomalley) [inline docs/cli-usage.md:1]: @pabloest Note that I don't include this in the navigation tree

Comment (pabloest) [reply docs/cli-usage.md:1]: I am fine with excluding it but how do I get to this doc in general?

File: docs/cli-usage.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+The following usage documentation is for the [Semgrep CLI](https://github.com/returntocorp/semgrep).
+
+[TOC]
```
Comment (pabloest) [inline docs/cli-usage.md:5]: TOC is probably not needed here? This doc is super short.

File: docs/cli-usage.md (hunk 1 above)
Comment (dlukeomalley) [reply docs/cli-usage.md:1]: It linked from the Semgrep README and appears in search results

File: docs/cli-usage.md (hunk 2 above)
Comment (dlukeomalley) [reply docs/cli-usage.md:5]: I've gotten into the practice of adding them. TOC provides anchor links too, should that be nice. Happy to remove though.

Comment (pabloest) [reply docs/cli-usage.md:5]: Ah yes, probably useful for the anchors alone. 👍

**Pull request merged**